--- a/src/main/java/com/badwolfmc/rolerewards/command/RoleRewardsCommand.java
+++ b/src/main/java/com/badwolfmc/rolerewards/command/RoleRewardsCommand.java
@@ -89,7 +89,7 @@
                                         context,
                                         StringArgumentType.getString(context, "reward"),
                                         StringArgumentType.getString(context, "period")
-                                ))))));
+                                )))));
 
         root.then(Commands.literal("reload")
                 .requires(source -> Permissions.has(source.getSender(), Permissions.RELOAD))
